app/controllers/api/integrations_controller.rb

```diff
@@ -243,6 +243,8 @@ def proxy_params
       basic_fields << :oidc_issuer_type
     end
 
+    basic_fields << { backend_api_configs_attributes: %i[_destroy id path] } if @service.act_as_product?
```

Comment:
This should be tested in `test/functional/api/integrations_controller_test.rb` 😄 But it can be done in another PR 👍

Reply:
Specially the `_destroy` is not obvious without looking at the rest of the code and it is easy to mess up if there is no test.

Reply:
Same as https://github.com/3scale/porta/pull/1008#discussion_r315135540